doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_mongo_sink.md

```diff
@@ -286,6 +287,16 @@ If `data_model=dm-by-entity` and `attr_persistence=column` then `NGSIMongoSink`
 
 [Top](#top)
 
+### <a name="section1.4"></a>MongoDB Indexes
+Since version 3.0.0 of cygnus an index named `cyg_raw_opt` is created in this way but depending on datamodel:
+
+| datamodel       | keys                                                             |
+| :-------------- | :--------------------------------------------------------------- |
+| DMBYSERVICEPATH | recvTime, entityId, entityType, attrName, attrType, attrValue    |
+| DMBYENTITY      | recvTime, attrName, attrType, attrValue                          |
+| DMBYATTRIBUTE   | recvTime, attrType, attrValue                                    |
```

Comment:
Maybe we should use the names that are used in configuration by users (i.e `dm-by-...`). I understand that the `DM...` being used are the ones from the code (so, somehow "hidden" by regular users).

In addition, what about the other DMs (from https://github.com/telefonicaid/fiware-cygnus/blob/master/doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_postgresql_sink.md#section1.2 it seems we have 11 DMs, but this table only includes 3)? If they don't involve indexes maybe we should add a sentence like this:

> In the case of other datamodels than the ones included in the table above (including the default behaviour when no datamodel is set) cygnus doesn't create any index.

(Same applies to the modification done in PR https://github.com/telefonicaid/fiware-sth-comet/pull/624. Maybe a follow up PR in that repo should be done)

Reply:
Thus, I'd suggest to add the following note just after the table in documentaiton:

> Note that attempting to use any other datamodel different from these ones will cause a "Unknown data model" ERROR in log traces. In addition, note that default datamodel for STH/Mongo sink is ...

completing the missing "..."